src/onboarding/steps.ts

```diff
@@ -77,10 +103,15 @@ export function onboardingPropsSelector(state: RootState): OnboardingProps {
  * and count the number of screens until the given screen and the total number
  */
 export function getOnboardingStepValues(screen: Screens, onboardingProps: OnboardingProps) {
+  const firstScreen = firstOnboardingScreen({
+    onboardingNameScreenEnabled: onboardingProps.onboardingNameScreenEnabled,
+    recoveringFromStoreWipe: onboardingProps.recoveringFromStoreWipe,
+  })
+
   let stepCounter = 1 // will increment this up to the onboarding step the user is on
   let totalCounter = 1
   let reachedStep = false // tracks whether we have reached the step the user is on in onboarding, and we can stop incrementing stepCounter
-  let currentScreen: Screens = FIRST_ONBOARDING_SCREEN // pointer that we will update when simulating navigation through the onboarding screens to calculate "step" and "totalSteps"
```

Comment:
I think the `FIRST_ONBOARDING_SCREEN` is unused now, not sure if you want to keep it.